docs-web/configuration/policy-resource.md

```diff
@@ -189,6 +191,53 @@ policies:
 
 When you reference more than one rate limit policy, the Ingress Controller will configure NGINX to use all referenced rate limits. When you define multiple policies, each additional policy inherits the `dryRun`, `logLevel`, and `rejectCode` parameters from the first policy referenced (`rate-limit-policy-one`, in the example above).
 
+### JWT
+
+> Note: The feature is only available in NGINX Plus.
+
+The JWT policy configures NGINX Plus to authenticate client requests using JSON Web Tokens.
+
+For example, the following policy will reject all requests that do not include a valid JWT in the HTTP header `token`:
+```yaml
+jwt:
+  secret: jwk-secret
+  realm: "My API"
+```
+
+> Note: The feature is implemented using the NGINX Plus [ngx_http_auth_jwt_module](https://nginx.org/en/docs/http/ngx_http_auth_jwt_module.html).
```

Comment:
I would combine this note with the one above about N+. It's good to put this type of information up front.

Reply:
I put the NGINX Plus requirement first because it's important for everyone to know.

However, this note is more about its implementation 'under the hood', for a minority of advanced users. So I don't really think it makes sense at the start. 

I think I'm going to leave it as it is for now. Let me know if you disagree and we can revisit this. Ideally making a consistent change across all policy docs.

Reply:
Sounds good to me, thanks for the rationale!